contracts/src/TeleporterTokenSource.sol

```diff
@@ -27,6 +29,22 @@ import {IWarpMessenger} from
  * DO NOT USE THIS CODE IN PRODUCTION.
  */
 
+/**
+ * @notice Each destination bridge instance registers with the source token bridge contract,
+ * and provides settings for bridging to the destination bridge.
+ * @param registered whether the destination bridge is registered
+ * @param collateralNeeded the amount of tokens that must be first added as collateral,
+ * through `addCollateral` calls, before tokens can be bridged to the destination token bridge.
+ * @param tokenMultiplier the scaling factor for the amount of tokens to be bridged to the destination.
+ * @param multiplyOnSend whether the scaling factor is multiplied or divided when sending to the destination.
+ */
+struct DestinationBridgeSettings {
+    bool registered;
```

Comment:
prefer to leave as is at this point, but that makes sense.

Reply:
It's a minor optimization on the contract's state since these instances are persisted in the mapping, but the mapping is likely to remain small for the lifetime of the contract anyway, so agreed readability is preferred.